src/benchmark/tests/executors_test.py

```diff
@@ -0,0 +1,112 @@
+import sys
```

Comment:
according to pytest guidelines file names should start with test_

Reply:
fixed in [c7f6d16](https://github.com/zmaslova/dl-benchmark/pull/14/commits/c7f6d16f528eb0f0f291444cdc34899205940c9e)